### ISSUE TITLE
Video not migrated in 1.0.9

### ISSUE THREAD

**Issue by StaceZ** [nextcloud/integration_google#128]
I synced my Google Photos, but all video files now have HTML content like:
```
# cat 6min-rotated.mp4
<HTML><HEAD><meta http-equiv="content-type" content="text/html;charset=utf-8">
<TITLE>302 Moved</TITLE></HEAD><BODY>
<H1>302 Moved</H1>
The document has moved
<A HREF="https://video-downloads.googleusercontent.com/AEpB0ooPxMTdswBGv8D9fsL-nIsjFxVOaxRCxuuqwgSu0s_1kmqDnlvLPp9GBFNfMZh-ZmkNAFeI0DBa0E8G8HSDwlCY_p4jM4f-X9T27vfwJSpdkBA9OkfIH9ELDEd59xa2_v_FNJl1RVetfBSDPeBxxWrLbh0q1RTvQMwpTy_1VC3glaBybSFNHXyq8NcnFtA4RQdhSskW4qokC1FE6OUtrYeEGw8M_JI1t6k5zUkQZOat2eTKOrEd1UAkReLkRr2I45A-QtZiMaYilJ3NC7QmAdeIKx35g1LQNPLsFOrnFp1sU1Cwnm5UAWDUSfw71jJ2sKWCxMo88zB0rjwWvIArqE50QNsDab89SDFukn753wh2t5w0k7yr-trNGK_18bcX48Dt1YrSbLxwitt7veAlIqBswMP3t9fEqHDh9cm559lk_34w3PHrk5cmeyN1hQbPXbS_4zHF5o2nUQKF83mU41xjl....">here</A>.
</BODY></HTML>
```
A year ago they synced successfully

**Comment by julien-nc** [comment]
Thanks for the bug report. I can't reproduce this issue. The internal HTTP client should follow the 302 response and correctly download the video. I don't know why this fails in your case. Nothing changed in this area as far as I see in the implementation.

Do you remember having changed anything that could be related in your server system's Php config or in Nextcloud's config.php?

**Comment by StaceZ** [comment]
NextCloud updated to 23.0.12.
PHP updated to 7.3.31-1~deb10u2 with Debian OS upgrade.
No other changes

**Comment by StaceZ** [comment]
I found interesting message: "the Google servers do reply with a 200 and not with a 302 when using the Symfony HttpClient. The reason is that Google does some User-Agent sniffing: forcing User-Agent: curl/7.58.0 on the request triggers a 302." https://github.com/symfony/symfony/issues/34821#issuecomment-567435844
Maybe this is my problem?

**Comment by StaceZ** [comment]
Problem fixed after upgrade to 2.0.2. Thanks
